student-os/scripts/materials_convert.py

```
@@ -722,6 +865,184 @@ def convert_with_mineru_chunked(
     return payload
 
 
+def effective_agent_chunk_pages(page_count: int, chunk_size: int) -> int:
+    return max(1, min(chunk_size, MINERU_AGENT_MAX_PAGES))
+
+
+def convert_with_mineru_agent_chunked(
```

Ruff (0.16.1)

[warning] Too many branches (22 > 12)

(PLR0912)